optaweb-employee-rostering-frontend/src/ui/components/WeekPicker.tsx

```diff
@@ -40,28 +35,71 @@ function getLastDayInWeek(dateInWeek: Date): Date {
   return moment(dateInWeek).endOf('week').toDate();
 }
 
-export default class WeekPicker extends React.Component<WeekPickerProps, WeekPickerState> {
-  constructor(props: WeekPickerProps) {
-    super(props);
-    this.state = { isOpen: false };
-  }
-
+export default class WeekPicker extends React.Component<WeekPickerProps> {
   goToCurrentWeek() {
     this.goToWeekContaining(new Date());
   }
 
   goToWeekContaining(date: Date) {
     if (this.props.minDate && getFirstDayInWeek(date) < getFirstDayInWeek(this.props.minDate)) {
       this.goToWeekContaining(this.props.minDate);
-    } else if (this.props.maxDate && getLastDayInWeek(date) > getLastDayInWeek(this.props.maxDate)) {
+      return;
+    } if (this.props.maxDate && getLastDayInWeek(date) > getLastDayInWeek(this.props.maxDate)) {
```

Comment:
`if` should start on a new line.

Reply:
I wonder why this isn't an eslint violation; I didn't get any errors or warnings when running `npm run lint`